criu/cgroup-props.c

```diff
@@ -319,6 +319,7 @@ static int cgp_parse_stream(char *stream, size_t len)
 					  (cgp_entry->cgp.nr_props + 1) * sizeof(char *))) {
 				pr_err("Can't allocate property for controller %s\n",
 				       cgp_entry->cgp.name);
+				xfree(p);
 				goto err_parse;
```

Comment:
Above in the same function we have two consequent `p = get_quoted` without any free, probably it's also a problem.

Reply:
I already had a closer look at this function and I think this is now correct. Once `p` is assigned to `cgp_entry->cgp.name` it will be cleaned up using `cgp_free()`.

Reply:
I missed it, then it looks good.